scriptmodules/ports/wolf4sdl.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# This file is part of RetroPie.
+# 
+# (c) Copyright 2012-2015  Florian Müller (contact@petrockblock.com)
+# 
+# See the LICENSE.md file at the top-level directory of this distribution and 
+# at https://raw.githubusercontent.com/petrockblog/RetroPie-Setup/master/LICENSE.md.
+#
+
+rp_module_id="wolf4sdl"
+rp_module_desc="Wolf4SDL, port of Wolfenstein 3D"
+rp_module_menus="4+"
+rp_module_flags="dispmanx"
+
+function depends_wolf4sdl() {
+    getDepends libsdl1.2-dev
+}
+
+function sources_wolf4sdl() {
+    local cmd=(dialog --clear --backtitle "$__backtitle" --menu "Choose which version of Wolfenstein 3D game data you wish to use in Wolf4SDL." 22 86 16)
+    local options=(
+	1 "Shareware 1.4" 
+	2 "3D Realms (Apogee 1.4)" 
+	3 "Steam (GT/ID/Activision)"
+    )
+    local choice=$("${cmd[@]}" "${options[@]}" 2>&1 >/dev/tty)
+        if [[ -n "$choice" ]]; then
+            gitPullOrClone "$md_build" https://github.com/mozzwald/wolf4sdl.git
+            case $choice in
+                1)
+                    sed -i '/GOODTIMES/d' version.h
+                    ;;
+                2)
+                    sed -i '/GOODTIMES/d' version.h
+                    sed -i '/UPLOAD/d' version.h
+                    ;;
+                3)
+                    sed -i '/UPLOAD/d' version.h
+                    ;;
+            esac
+        else
+            break
+        fi
+}
+
```

Comment:
There can't be user input in the sources part - wont be compatible with out binaries later etc - The best thing to do would be to build all three versions separately.